src/app/listing/listing.page.html

```diff
@@ -28,7 +28,7 @@ <h1>{{stateData?.name}}</h1>
   <ion-grid>
     <ion-row>
       @for (item of listData; track item.id) {
```

Comment:
it is recommended to name variables which highlights what it is about instead of general names like data, item, Please check if that can be done (optional)

Reply:
done